documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -147,19 +139,125 @@ Import the following into PGAdmin
+
+### Run the Backend API
+
+- check which version of Node you should be using.
+
+```bash
+ cd ./backend
+ cat package.json | less
+ # your version should look like this:
+...
+  "engineStrict": true,
+  "engines": {
+    "node": ">=18.18.0"
+  },
+...
+```
+
+- Switch to the correct (currently: v18.18.0) Node version:
+
+```bash
+cd ./backend
+nvm list # find out what versions are available for you
+nvm install  v18.18.0 # install matching version from package.json
+nvm use v18.18.0 # switch to that version
+nvm alias default v18.18.0 # Optional: set the default version
+npm i # installs dependencies
+npm run start # runs the backend API
+```
+
+---
+
+---
+
+## Frontend Setup and Deploy
+
+### Set up Frontend App
+
+- Create`./frontend/.env` using the example below as a template:
+- the `REACT_APP_KEYCLOAK_URL`, `REACT_APP_KEYCLOAK_CLIENT_ID`, `REACT_APP_KEYCLOAK_REALM` will have to be updated according to your [keycloak server settings](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/acd38d-dev/deployments/pmo-app-deployment/yaml)
+
+```bash
+# frontend .env
+WDS_SOCKET_PORT=3000
+REACT_APP_API_URL="https://localhost:8080/"
+REACT_APP_KEYCLOAK_CLIENT_ID=# see note above for value
+REACT_APP_KEYCLOAK_REALM=standard
+REACT_APP_KEYCLOAK_URL=# see note above for value
+```
+
+- check which version of Node you should be using.
+
+```bash
+ cd ./frontend
+ cat package.json | less
+ # your version should look like this:
+...
+  "engineStrict": true,
+  "engines": {
+    "node": ">=18.18.0"
+  },
+...
+```
+
+- Switch to the correct (currently: v18.18.0) Node version:
+
+```bash
+cd ./frontend
+nvm list # find out what versions are available for you
+nvm install  v18.18.0 # install matching version from package.json
+nvm use v18.18.0 # switch to that version
+nvm alias default v18.18.0 # Optional: set the default version
+npm i # installs dependencies
+npm run start # runs the frontend App
+```
+
+- Your browser should open up at [localhost:3000](https://localhost:3000)
+
+### Deploy the Application After Setup
+
+```bash
+cd ./backend
+docker compose up # starts database container
+npm run start # starts API backend
+cd ../frontend
+npm run start # starts APP frontend
+```
+
+---
+
+---
+
+### APPENDIX: How to reset local database using PGAdmin
+
+- make sure the db container is active:
+
+```bash
+cd ./backend
+docker ps # If the database container is not running, do:
+docker-compose up db
+```
 
-- If the database container is not running, go to the `./backend` folder and run ```docker-compose up db```
-- In pgAdmin, right click the pmo database and drop/delete it.
-- In pgAdmin, right click your pmo server and create a new database with the same name as the old pmo database.
\ No newline at end of file
+- In pgAdmin:
+  - Right click the GDX Agreements Tracker server > Remove Server
```

Comment:
@ShawnTurple maybe I misunderstood this part. 

was the goal to actually delete the `data` schema entirely, or was it to just delete and restore the database config from PGAdmin?

Reply:
i believe this was to delete it completely

Reply:
ok i'll update my instructions then

Reply:
done!